Documentation/getting-started.md

```diff
@@ -44,10 +44,11 @@ The Edge Network extension depends on the following extensions:
 
 1. Add the Mobile Core, Identity for Edge Network, and Edge Network extensions to your project using the app's Gradle file:
 
-```gradle
-implementation 'com.adobe.marketing.mobile:core:2.+'
-implementation 'com.adobe.marketing.mobile:edge:2.+'
-implementation 'com.adobe.marketing.mobile:edgeidentity:2.+'
+```kotlin
+implementation(platform("com.adobe.marketing.mobile:sdk-bom:3.+"))
+implementation("com.adobe.marketing.mobile:core")
+implementation("com.adobe.marketing.mobile:edge")
+implementation("com.adobe.marketing.mobile:edgeidentity")
```

Comment:
can we remove the parenthesis and use a format similar to our documented examples - https://developer.adobe.com/client-sdks/resources/migration/android/migrate-to-2x/#gradle

Reply:
As we changed our build files to use Kotlin, I changed them from the Groovy format to the Kotlin format. Android is also asking users to migrate to Kotlin, https://developer.android.com/build/migrate-to-kotlin-dsl#:~:text=Kotlin%20is%20preferred%20over%20the,time%20checking%20and%20IDE%20support.

Reply:
do we need to indicate the format used somewhere, and do we need to document both the formats from now on? cc @praveek